slices/erlang-inets.yaml

```diff
@@ -0,0 +1,26 @@
+package: erlang-inets
+
+essential:
+  - erlang-inets_copyright
+
+slices:
+  bins:
+    essential:
+      - erlang-base_bins
```

Comment:
I think they depend on the bins because those are elixir modules that needs the erlang binaries to run.

Reply:
It seems to me that the `erlang-inets` package contains mostly libraries (.beam files etc.) which should not, in my opinion, depend on any erlang-base binaries. Will you please double-check and let me know if (and how) they are needed? Thanks!

Reply:
the dependency here is because those beam files cannot be run standalone without erlang binaries found in erlang-base package. So if someone installed this slice without erlang-base being installed along, they'll end up with module files that they cannot run.

Reply:
I've made similar comments above.

I guess the problem here is that you're including beam files in the `libs` slice, which is odd cause they are closer to executables than they are to libraries. And you're also mixing them with header files.

So, If you're sure the beam files need the erlang-base binaries, the dependency should stay. Regardless of that, the `libs` aren't suited for these cases.